src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(operation, number) {
+      operation(number);
+
+      return this;
+    },
+
+    add(x) {
+      calculator.result += x;
+    },
+
+    subtract(x) {
+      calculator.result -= x;
+    },
+
+    divide(x) {
+      calculator.result /= x;
```

Comment:
Within the `divide` method, `calculator.result` should be `this.result` to correctly refer to the result property of the current calculator instance. You also need to add a check for 0, since you cannot divide by 0